src/services/API/RootClient.js

```diff
@@ -42,9 +60,40 @@ client.interceptors.request.use(
 // handle errors in responses for API requests
 client.interceptors.response.use(
   (config) => {
+    // cancel the axios response if the user is not logged in
+    // and do it only for the requests that are not related to authentication
+    if (
+      !store.getters["auth/isAuthenticated"] &&
+      !authEndpoints.includes(config.config.url)
+    )
+      throw new axios.Cancel("User is not logged in");
```

Comment:
i don't think I got the explanation for this + string translation was missed out here.